### DISCUSSION TITLE
荣耀 文件管理 无法获取id

### DISCUSSION THREAD

**Discussion by windwhim** [gkd-kit/gkd#739] [Q&A]
![Screenshot_20240920_134842_com_android_packageinstaller_AboutActivity.jpg](https://github.com/user-attachments/assets/4192bbea-0c85-4c88-bad6-cbb5fb9ce7ce)

在荣耀的文件管理应用中悬浮窗不显示(autojs同样如此)，该如何获取其中的元素id呢。

**Comment by lisonge** [comment]
https://gkd.li/guide/snapshot#capture-snapshot
